packages/web/src/components/spa/MainPage.jsx

```diff
@@ -12,9 +12,9 @@ import { checkHashAndScroll } from '../../Global';
 
 const LandingHome = () => {
     const anchorList = [
-        new Anchor('About', '#AboutSection'),
+        new Anchor('About', '#about'),
         new Anchor('Team', '#team'),
```

Comment:
As mentioned above, please make sure the About section anchor for landing page is connected correctly.